### DISCUSSION TITLE
Backup Release Artifacts

### DISCUSSION THREAD

**Discussion by 42CrMo4** [cooperspencer/gickup#180] [Q&A]
Hello, 
first of all thank you for the awsome tool, it works wonderfully. 
I just have one question. Is it possible to backup release artifacts from github to gitea? Have I missed this feature or is it not possible? I have tried searching the documentation and open and closed issues for similar things. 
Thank you very much.

**Comment by cooperspencer** [comment]
Hi,
As of now, this is not possible, but I can conduct some research and implement it.

**Comment by 42CrMo4** [comment]
Thank you very much for getting back to me so quickly. I would be very grateful for your efforts and would appreciate it if this was possible.

**Reply by cooperspencer** [reply]
Gitea doesn't let me mirror releases. If I migrate a repository without a mirror it is fine, but as soon as I try to run it as mirror it doesn't work. I'll play around a bit more, maybe I can do it with some coding effort.

**Reply by 42CrMo4** [reply]
Too bad this is not possible via the mirror feature. Probably the only way then is via the api, but I do not know if that is feasible.
